crates/observe/src/tracing.rs

```diff
@@ -25,22 +31,29 @@ pub fn initialize_reentrant(env_filter: &str) {
 }
 
 fn set_tracing_subscriber(env_filter: &str, stderr_threshold: LevelFilter) {
-    // This is what kibana uses to separate multi line log messages.
-    let subscriber_builder = tracing_subscriber::fmt::fmt()
+    let fmt_layer = tracing_subscriber::fmt::layer()
```

Comment:
The documentation seems to indicate that this can be enabled by just adding:

> If you are manually configuring the `tracing` subscriber using the [`EnvFilter`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html) or [`Targets`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/targets/struct.Targets.html) filters from [`tracing-subscriber`](https://crates.io/crates/tracing-subscriber), add `"tokio=trace,runtime=trace"` to your filter configuration.

Might be easier to go this route instead of setting up custom tracing subscriber layers.

**That being said**, I prefer what you did here with setting up an additional `console_subscriber` layer when the configuration option is enabled.